### PR TITLE
added verification feature

### PR DESCRIPTION
**Pull request by Aram-Abovyan**

### CODE REVIEW

File: GG_clone_backend/db/seeds.rb [hunk 1]
```diff
@@ -5,25 +5,63 @@
...
+#                 password_digest: Driver.digest('Driver2') })
+# Driver.create({ first_name: 'Hambardzum', last_name: 'Bakunts', phone_number: '+37455896532',
+#                 email: 'bambardzum1298@mail.com', car_model: 'Vectra', car_level: 'econom', car_manufacturer: 'Opel',
+#                 car_registration_number: '19de966', is_active: true, is_verified_by_admin: true,
+#                 password_digest: Driver.digest('Driver3') })
 
-SuperUser.create(phone_number: '+37441322020', first_name: 'Vazgen', last_name: 'Admin',
-                 password_digest: SuperUser.digest('Admin1234'))
+# SuperUser.create(phone_number: '+37441322020', first_name: 'Vazgen', last_name: 'Admin',
+#                  password_digest: SuperUser.digest('Admin1234'))
```
Comment (vnersisyan-vineti) [inline GG_clone_backend/db/seeds.rb]: same comment as in previous PR, if not used remove comment part

File: src/components/Admin/ClientFieldDriver.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 import PropTypes from 'prop-types';
+import VerifyButton from './verification/VerifyButton';
 
 const ClientFieldDriver = ({
   driver: {
     id, phoneNumber, firstName, lastName, isActive, isVerifiedByAdmin,
   },
+  onClick,
 }) => {
   let verified = 'bg-danger';
```
Comment (vnersisyan-vineti) [inline src/components/Admin/ClientFieldDriver.js]: rename verified variable name to `verifiedClassName`
`const verifiedClassName = isVerifiedByAdmin ? 'bg-success' : 'bg-danger';
`const activeClassName = isActive && isVerifiedByAdmin ? 'bg-success' : 'bg-danger';

File: src/components/Admin/verification/UnverifiedDriverList.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  const handleVerify = (id) => () => {
+    axios.defaults.headers.post['Content-Type'] = 'application/x-www-form-urlencoded';
+    axios.defaults.headers.post.Accept = '*/*';
+
+    axios.post(UNVERIFIED_URL,
+      { id },
+      { withCredentials: true })
+      .then((res) => {
+        if (res.statusText === 'OK') {
+          setUnverifiedDrivers(unverifiedDrivers.filter((driver) => driver.id !== id));
```
Comment (vnersisyan-vineti) [inline src/components/Admin/verification/UnverifiedDriverList.js:34]: separate new drivers to variable, then pass it to `setUnverifiedDrivers`

File: src/components/Admin/verification/UnverifiedDriverList.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+  useEffect(() => {
+    axios.defaults.headers.get.Accept = '*/*';
+
+    axios.get(UNVERIFIED_URL,
+      {
+        'Accept-Encoding': 'gzip, deflate, br',
+        withCredentials: true,
+      })
+      .then((res) => {
```
Comment (vnersisyan-vineti) [inline src/components/Admin/verification/UnverifiedDriverList.js]: `({ data })` - do destruction in argument section

File: src/components/Admin/verification/validateDriver.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import camlecase from 'camelcase';
+
+export default function validateDriver(driver) {
+  return Object.fromEntries(Object.entries(driver).map((item) => [camlecase(item[0]), item[1]]));
```
Comment (vnersisyan-vineti) [inline src/components/Admin/verification/validateDriver.js]: `.map((item) => [camlecase(item[0]), item[1]]));` to not have array indexes here, destruct in map params section, `.map([name1, name2])`, and also name those variables with field names

**Review by vnersisyan-vineti — CHANGES_REQUESTED** [review]
Minor comments, please address

File: GG_clone_backend/db/seeds.rb [hunk 6]
```diff
@@ -1,29 +1,38 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
-Passenger.create({ first_name: 'Hakob', last_name: 'Arshakyan', phone_number: '+37444556699',
```
Comment (iwazowsky) [inline GG_clone_backend/db/seeds.rb:8]: This manual seeds are required for testing, please keep them

File: GG_clone_backend/app/controllers/api/v1/unverified_drivers_controller.rb [hunk 7]
```diff
@@ -0,0 +1,11 @@
+class Api::V1::UnverifiedDriversController < ApplicationController
```
Comment (ArtyomKosakyan) [inline GG_clone_backend/app/controllers/api/v1/unverified_drivers_controller.rb:1]: On my opinion we can write this functionality in drivers_controller for example, but we don't have time, dont change

File: src/components/Admin/ClientField.js [hunk 8]
```diff
@@ -1,12 +1,19 @@
-import React from 'react';
+/* eslint-disable */
```
Comment (ArtyomKosakyan) [inline src/components/Admin/ClientField.js:1]: Why Eslint disable ?

File: src/components/Admin/ClientFieldDriver.js [hunk 9]
```diff
@@ -1,35 +1,40 @@
+/* eslint-disable */
```
Comment (ArtyomKosakyan) [inline src/components/Admin/ClientFieldDriver.js:1]: Why disabled?

**Review by ArtyomKosakyan — APPROVED** [review]

**Review by vnersisyan-vineti — APPROVED** [review]
Approve

**Pull request merged**